### PR TITLE
Fix/idea detail desktop

### PR DESCRIPTION
**Pull request by goyadev**
- Fixed buttons to incl new button styles
- added hardcoded features as per Tue night review discussion
- added desktop and tablet responsiveness

To do next: 
- add code for counting down EOIs and tracking through buttons
- figure out how to get images to happen

Screenshot 2024-04-04 at 4 53 38 pm
Screenshot 2024-04-04 at 4 53 25 pm
Screenshot 2024-04-04 at 4 53 14 pm

### CODE REVIEW

File: src/pages/IdeaDetail.jsx [hunk 1]
```diff
@@ -1,10 +1,27 @@
+import { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
 import useWorkshop from "../hooks/use-workshop";
 import Button from "../components/Button";
 
 export default function LoginPage() {
```
Comment (suanTech) [inline src/pages/IdeaDetail.jsx]: Please change the name to IdeaDetail

**Review by suanTech — COMMENTED** [review]
There's a small type error!

File: src/pages/IdeaDetail.jsx (hunk 1 above)
Comment (goyadev) [reply src/pages/IdeaDetail.jsx]: On it! I fixed it in one version, then forgot to update it

**Review by suanTech — APPROVED** [review]
sweeet ❤️‍🔥

**Pull request merged**